src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/Cas2AutoScriptTest.kt

```diff
@@ -101,13 +103,16 @@ class Cas2AutoScriptTest {
 
     every { mockApplicationService.createCas2ApplicationSubmittedEvent(any()) } answers { }
     every { mockStatusUpdateService.createStatusUpdatedDomainEvent(any()) } answers { }
+    mockkConstructor(Cas2SeedUtils::class)
```

Comment:
I'm not sure why I had to do this to mock the `insertHdcDates` - I would have thought a regular `mockk<Cas2SeedUtils>()` would have worked but it wasn't picking up the mock. Maybe because it's just a regular class that gets instantiated, whereas the other things we mock are annotated beans??